regression/CMakeLists.txt

```diff
@@ -20,6 +20,20 @@ macro(add_test_pl_tests cmdline)
     add_test_pl_profile("${TEST_DIR_NAME}" "${cmdline}" -K KNOWNBUG ${ARGN})
 endmacro(add_test_pl_tests)
 
+# Standalone regression tests that do not need test.pl
+macro(add_standalone_test name cmdline)
+  add_test(
+    NAME "${name}-CORE"
+    COMMAND "${cmdline}"
```

Comment:
Note this still doesn’t quite work like it should – this is just the first parameter passed to the macro, which is just going to be `python3`. You’ll want to remove the `cmdline` parameter completely and just use `${ARGN}` (i.e. the remainder of the arguments after the named ones) here, otherwise the other arguments (i.e. `test.py` in your case) won’t be passed on the command line.